server.js

```diff
@@ -1,15 +1,44 @@
 const express = require('express')
 const bodyParser = require('body-parser')
 
-const { buildCommitsMessage, fetchUnreleasedCommits } = require ('./utils/commits')
-const { postToSlack } = require('./utils/commits')
+const { buildUnreleasedCommitsMessage, fetchUnreleasedCommits } = require ('./utils/unreleased-commits')
+const { buildUnmergedCommitsMessage, fetchUnmergedPRs } = require('./utils/unreleased-commits')
+const { postToSlack } = require('./utils/commits-helpers')
 
 const app = express()
 app.use(bodyParser.urlencoded({ extended: true }));
 app.use(bodyParser.json())
 
 app.use(express.static('public'))
 
+app.post('/unmerged', async (req, res) => {
```

Comment:
i haven't added this endpoint to the Slack side of things yet, still need to do that as well as add a new slack command